Solr drupal_published not found

Hi, thanks for the previous help.

I followed all the instructions, but when I go to access from the browser,
I have the follow error message:

> "The website encountered an unexpected error. Please try again later.</br></br><em class="placeholder">Drupal\search_api_solr\SearchApiSolrException</em>: Solr endpoint http://localhost:8983/solr/drupal_published/ not found. in "

At the step of execute:  ./vendor/bin/phing setup-apache-solr, I had problem,
for these reason I installed solr for my own.  Had the solr included in the project, special configurations?

The Error when I execute  ./vendor/bin/phing setup-apache-solr, the problem is that
 http://www-eu.apache.org/dist/lucene/solr/6.6.0/solr-6.6.0.tgz is not available.

The trace  is:

> [root@desa-arq-01 joinup-dev]# ./vendor/bin/phing setup-apache-solr
> Buildfile: /usr/local/bin/joinup-dev/build.xml
>    [import] Unable to find build file: /usr/local/bin/joinup-dev/build.local.xml... skipped
>      [echo] Loading AWS additional tasks.
>      [echo] Turn environment variable into a phing property
>      [echo] Loading infra additional tasks.
>  [property] Loading /usr/local/bin/joinup-dev/build.properties.local
>  [property] Loading /usr/local/bin/joinup-dev/build.properties
>  [property] Loading /usr/local/bin/joinup-dev/build.properties.dist
>      [echo] Loading Behat task.
>      [echo] Loading After Import RDF fixtures task.
>      [echo] Setting-up MySQL auto-increment values for {comment}, {file_managed}, {node} and {users}.
>      [echo] Loading Amazon S3 download task.
>      [echo] Loading Amazon S3 list task.
>      [echo] Loading Truncate Cache Tables task.
>      [echo] Loading Empty MySQL Database task.
>      [echo] Loading Drush task.
>      [echo] Loading PHP Codesniffer Configuration task.
>      [echo] Loading PHPUnit Configuration task.
>  [property] Loading /usr/local/bin/joinup-dev/build.properties.local
>  [property] Loading /usr/local/bin/joinup-dev/build.properties
>  [property] Loading /usr/local/bin/joinup-dev/build.properties.dist
> 
> My Drupal Project > download-apache-solr:
> 
>   [httpget] Fetching http://www-eu.apache.org/dist/lucene/solr/6.6.0/solr-6.6.0.tgz
>        [if] Error in IfTask
> 
> BUILD FAILED
> /usr/local/bin/joinup-dev/build.solr.xml:6:12: /usr/local/bin/joinup-dev/build.solr.xml:12:20: Error in IfTask
> #0 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/UnknownElement.php(100): TaskAdapter->main()
> #1 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Task.php(283): UnknownElement->main()
> #2 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/SequentialTask.php(58): Task->perform()
> #3 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/IfTask.php(186): SequentialTask->main()
> #4 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/TaskAdapter.php(61): IfTask->main(Object(Project))
> #5 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/UnknownElement.php(100): TaskAdapter->main()
> #6 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Task.php(283): UnknownElement->main()
> #7 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Target.php(336): Task->perform()
> #8 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Target.php(366): Target->main()
> #9 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Project.php(898): Target->performTasks()
> #10 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Project.php(868): Project->executeTarget('setup-apache-so...')
> #11 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(723): Project->executeTargets(Array)
> #12 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(200): Phing->runBuild()
> #13 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(347): Phing::start(Array, NULL)
> #14 /usr/local/bin/joinup-dev/vendor/phing/phing/bin/phing.php(58): Phing::fire(Array)
> #15 /usr/local/bin/joinup-dev/vendor/phing/phing/bin/phing(14): require_once('/usr/local/bin/...')
> #16 {main}
> 
> Previous HTTP_Request2_ConnectionException: Unable to connect to tcp://www-eu.apache.org:80. Error: php_network_getaddresses: getaddrinfo failed: Name or service not known in /usr/local/bin/joinup-dev/vendor/pear/http_request2/HTTP/Request2/Adapter/Socket.php on line 331
> #0 /usr/local/bin/joinup-dev/vendor/pear/http_request2/HTTP/Request2/Adapter/Socket.php(331): HTTP_Request2_SocketWrapper->__construct('tcp://www-eu.ap...', 10, Array)
> #1 /usr/local/bin/joinup-dev/vendor/pear/http_request2/HTTP/Request2/Adapter/Socket.php(128): HTTP_Request2_Adapter_Socket->connect()
> #2 /usr/local/bin/joinup-dev/vendor/pear/http_request2/HTTP/Request2.php(946): HTTP_Request2_Adapter_Socket->sendRequest(Object(HTTP_Request2))
> #3 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/ext/HttpTask.php(187): HTTP_Request2->send()
> #4 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/UnknownElement.php(100): HttpTask->main()
> #5 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Task.php(283): UnknownElement->main()
> #6 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/SequentialTask.php(58): Task->perform()
> #7 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/IfTask.php(186): SequentialTask->main()
> #8 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/TaskAdapter.php(61): IfTask->main(Object(Project))
> #9 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/UnknownElement.php(100): TaskAdapter->main()
> #10 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Task.php(283): UnknownElement->main()
> #11 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/SequentialTask.php(58): Task->perform()
> #12 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/tasks/system/IfTask.php(186): SequentialTask->main()
> #13 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/TaskAdapter.php(61): IfTask->main(Object(Project))
> #14 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/UnknownElement.php(100): TaskAdapter->main()
> #15 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Task.php(283): UnknownElement->main()
> #16 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Target.php(336): Task->perform()
> #17 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Target.php(366): Target->main()
> #18 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Project.php(898): Target->performTasks()
> #19 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Project.php(868): Project->executeTarget('setup-apache-so...')
> #20 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(723): Project->executeTargets(Array)
> #21 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(200): Phing->runBuild()
> #22 /usr/local/bin/joinup-dev/vendor/phing/phing/classes/phing/Phing.php(347): Phing::start(Array, NULL)
> #23 /usr/local/bin/joinup-dev/vendor/phing/phing/bin/phing.php(58): Phing::fire(Array)
> #24 /usr/local/bin/joinup-dev/vendor/phing/phing/bin/phing(14): require_once('/usr/local/bin/...')
> #25 {main}


Thanks for the help!

--- a/build.properties
+++ b/build.properties
@@ -131,7 +131,7 @@ phpunit.migrate.legacy_db.name = d6_joinup_test
 # -------------------------
 
 # Apache Solr version to use.
-solr.version = 6.6.0
+solr.version = 6.6.5
 
 # Filename of the download.
 solr.tarball.filename = solr-${solr.version}.tgz

